app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -185,7 +185,7 @@ export const ViewContent: React.FC<IViewContentProps> = ({ setActiveContent }) =
           <div className="source-section">{`${content?.section} ${
             content?.page && `:${content.page}`
           }`}</div>
-          {content?.tonePools && (
+          {content?.tonePools && content?.tonePools.length && (
```

Comment:
Fixed bug unrelated to this PR.  Developers please don't assume an array has items in it.